ONSdigital/eq-author-app
# Added Select to Storybook + made template

### What is the context of this PR?

Using content linked on the Jira ticket, I have created a template for documenting our components in Storybook using the Select component as an example. I have also swapped the Buttons over to this format and left the template empty for content.

### How to review 

Run `yarn && yarn storybook` to view the design system locally. Take a look and leave feedback.

* Import the **Capability examples** questionnaire from pre-prod Author into your local environment and:
    * ensure it can be opened in Author;
    * then, ensure it can be viewed in Runner by pressing the **view survey** button
* Does this require a migration? We need one if existing JSON schema properties change

### What to do after everything is green

1. * [ ] Bring this branch up-to-date with Origin/Master
2. * [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. * [ ] Click the **Merge** button on this pull request
4. * [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. * [ ] Move the Trello card for this task into the next stage of the process
